config/src/main/kotlin/org/springframework/security/config/annotation/web/AuthorizeHttpRequestsDsl.kt

```diff
@@ -64,6 +66,18 @@ class AuthorizeHttpRequestsDsl : AbstractRequestMatcherDsl() {
         authorizationRules.add(MatcherAuthorizationManagerRule(matches, access))
     }
 
+    /**
+     * Adds a request authorization rule by dispatch type
+     *
+     * @param dispatcherType the [DispatcherType] to match incoming requests against.
+     * @param access the [AuthorizationManager] to secure the matching request
+     * (i.e. created via hasAuthority("ROLE_USER"))
+     */
+    fun authorize(dispatcherType: DispatcherType,
```

Comment:
We should not add this new method since there is the option to do `authorize(DispatcherTypeRequestMatcher(DispatcherType.FORWARD), permitAll)` and the PR should only fix the documentation issue